Prose/ProseLib/Sources/SidebarFeature/SidebarView.swift

```diff
@@ -5,179 +5,80 @@
 //  Created by Valerian Saliou on 11/15/21.
 //
 
+import AppLocalization
 import ComposableArchitecture
-import ProseCoreStub
 import SwiftUI
 
-// MARK: - View
-
-/// The sidebar on the left of the screen.
 public struct SidebarView: View {
-    public typealias State = SidebarState
-    public typealias Action = SidebarAction
+    public typealias ViewState = SidebarState
+    public typealias ViewAction = SidebarAction
```

Comment:
You talked about specific `ViewState`s today.
Shouldn't we keep the separation between the `typealias` I named `State` for the `Store` and `ViewState` for the `ViewStore`s?

Reply:
I can remove the typealias if you'd prefer that. I don't think we need 4 typealiases if we have a Store and a ViewStore.

Reply:
I didn't mean to remove them, just keep the same convention as in other files, and I believe `State` for the `Store` and `ViewState` for the `ViewStore`s is quite logical, isn't it?

(We don't need 4, we can have only `ViewState` and `ViewAction`, but as long as they are used for a `ViewStore`)

Reply:
Actually I don't agree. There's only ViewState and ViewAction. Unless you create a `ViewState` struct or `ViewAction` enum which you scope your Store to, your store contains the `ViewState` and `ViewAction` because it is the same.

Reply:
I give in.

Reply:
Makes sense, you're right

Reply:
It's not a clear distinction and the names of the generic parameters Store and ViewStore are both the same (State and Action). They're using State and Action as typealiases in the isowords project as well, so I'll follow suite.